pkgs/applications/networking/gns3/gui.nix

```diff
@@ -33,25 +32,24 @@ python3.pkgs.buildPythonPackage rec {
     sentry-sdk
     setuptools
     sip_4 (pyqt5.override { withWebSockets = true; })
+    truststore
+  ];
+
+  pythonRelaxDeps = [
+    "jsonschema"
+    "sentry-sdk"
   ];
 
   doCheck = false; # Failing
 
   dontWrapQtApps = true;
 
   postFixup = ''
-      wrapQtApp "$out/bin/gns3"
-  '';
-
-  postPatch = ''
-    substituteInPlace requirements.txt \
-      --replace "psutil==" "psutil>=" \
-      --replace "jsonschema>=4.17.0,<4.18" "jsonschema" \
-      --replace "sentry-sdk==1.10.1,<1.11" "sentry-sdk"
+    wrapQtApp "$out/bin/gns3"
```

Comment:
Could you double check whether that's necessary? I thought wrapQtAppsHook would take care of this automatically, but I could be wrong here.

Reply:
It is required because `$out/bin/gns3` is non-ELF executable.

> wrapQtAppsHook ignores files that are non-ELF executables. This means that scripts won’t be automatically wrapped so you’ll need to manually wrap them as previously mentioned. An example of when you’d always need to do this is with Python applications that use PyQt.

See https://nixos.org/manual/nixpkgs/stable/#qt-runtime-dependencies